packages/shadydom/src/patch-events.js

```diff
@@ -305,7 +305,7 @@ function fireHandlers(event, node, phase) {
   }
 }
 
-function retargetNonBubblingEvent(e) {
+function shadyDispatchEvent(e) {
```

Comment:
Maybe needs bikeshedding, the non-bubbling part of `retargetNonBubblingEvent` isn't correct anymore though.

Reply:
Got it.